docs/Secure-Coding-Guide-for-Python/contribution/README_TEMPLATE.md

```diff
@@ -0,0 +1,78 @@
+# CWE-000: Title goes here
+
+Introduction paragraph...
```

Comment:
# CWE-000: Title goes here

Introduction sentence, this will be displayed in search engines.

Introduction paragraph, expanding on the introduction sentence